apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -502,11 +507,6 @@ namespace MWRender
 
         mRootNode->getOrCreateStateSet()->addUniform(new osg::Uniform("simpleWater", false));
 
-        // Hopefully, anything genuinely requiring the default alpha func of GL_ALWAYS explicitly sets it
-        mRootNode->getOrCreateStateSet()->setAttribute(Shader::RemovedAlphaFunc::getInstance(GL_ALWAYS));
```

Comment:
The state attribute here isn't related to the renderbin state. Without this at the root of the scenegraph, OSG makes an unnecessary `glAlphaFunc` call when drawing anything with no explicit alpha function having previously drawn something that did have one, thinking it has to restore the default state. Rendering will be the same as testing's off, but there'll be more deprecated function warnings in an OpenGL debugger.